.circleci/config.yml

```diff
@@ -233,17 +236,20 @@ jobs:
             make integration-tests
 
       #################################################################
-      # Run frontend tests. (Only executed in the Python 3 job.)
+      # Run frontend tests. (Only executed in one job.)
       #################################################################
       - run:
           name: Run frontend tests
           command: |
-            make jstest
+            if [ "${CIRCLE_JOB}" != "python-3.8" ] ; then
```

Comment:
Should we do 3.7.4 here since it runs first and there's no point in running the other two if the first fails? Or what's the thinking for doing it in 3.8?

Reply:
The three Python versions all run in parallel. I expect that we'll remove testing on 3.7.4 once we have this in and Tensorflow supports 3.8 as testing against the oldest and newest versions of Python we support should be sufficient. I just proactively moved this to the latest version we're testing against.